<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPUs</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>List of CPUs</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetCPUs">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>501</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Product Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Product Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CPU Socket</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Speed</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string># of Cores</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
